include/aspect/newton.h

```diff
@@ -61,10 +61,37 @@ namespace aspect
    * A Class which can declare and parse parameters and creates
    * material model outputs for the Newton solver.
    */
+
   template <int dim>
   class NewtonHandler: public SimulatorAccess<dim>
   {
     public:
+
+      /**
+       * This enum describes the type of stabilization is used
+       * for the Newton solver. None represents no stabilization,
+       * SPD represent that the resulting matrix is make Symmetric
+       * Positive Definite, symmetric represents that the matrix is
+       * only symmetrized, and PD represents that we do the same as
+       * what we do for SPD, but without the symmetrization.
+       */
+      enum class NewtonStabilization
```

Comment:
I blanked -- `enum class` is only C++11, but we can't presume that yet for ASPECT. Can you just make it `enum`? I think everything should continue to work anyway without further modifications.